### PR TITLE
test zen download box-draft

### PR DESCRIPTION
**Pull request by Himani1519**

### CODE REVIEW

**Comment by 1000TurquoisePogs** [comment]
This looks good but I am going to turn it into a draft for now because zen is not released yet so this should not be merged until then.

**Comment by balhar-jakub** [comment]
Wouldn't it make more sense for the box to be on the client side as the software is installed on the client side?

File: _data/releases.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ future:
       release_date: 2024-01-08
 
 v2:
-  - version: 2.12.0
+  - zen_version: 1.0.0
+    version: 2.12.0
```
Comment (1000TurquoisePogs) [inline _data/releases.yml:17]: i suspect you need to update your branch.

File: _includes/legal-script.html [hunk 2]
```diff
@@ -125,7 +125,56 @@
...
                 .get(
                     'package') + '.zip';
-        } else {
+        }
+        else if (params.has('type') && params.get('type') == "zen-windows") {
+            document.getElementById('download_button').href = "post_download.html?version=" + params.get('version') + "&type=" + params.get('type');
+            document.getElementById('download_button').onclick = function () {
+                ga && ga('send', 'event', 'download', 'zen-windows' + params.get('version'), 'zowe-enterprise-necessity-' + params.get(
+                    'version') + '.exe');
+                 window.open("https://zowe.jfrog.io/artifactory/libs-snapshot-local/org/zowe/zen/" + params.get('version') + '-V2.X/STAGING-166' +
```
Comment (1000TurquoisePogs) [inline _includes/legal-script.html:134]: This should work but should also change.
Talk with James about getting a better pattern for `libs-release-local` when we switch from `libs-snapshot-local` to it.

File: _includes/post-download-script.html [hunk 3]
```diff
@@ -28,6 +28,26 @@
...
+                package_name = 'zowe-install-wizard'
+                break;
+            case 'zen-mac':
+                filename_prefix = 'zowe-install-wizard-mac';
+                file_ext = '-x64.dmg';
+                package_name = 'zowe-install-wizard'
+                break;
+            case 'zen-rpm':
+                filename_prefix = 'zowe-install-wizard-rpm';
+                file_ext = '-1.x86_64.rpm';
```
Comment (1000TurquoisePogs) [inline _includes/post-download-script.html:43]: I'm not sure why the -1 is in the filename, but just filter on x86_64.rpm please in case it changes.

**Comment by 1000TurquoisePogs** [comment]
> Wouldn't it make more sense for the box to be on the client side as the software is installed on the client side?

@balhar-jakub two things,
1) since ZAC said "tech preview", i think it has to go in the bottom of the download page, which doesn't distinguish between client/server.

2) if it weren't go to in "tech preview", previous versions of this PR had it put in the "server" section.
Yes, it is a client-installed program, but its purpose is for server content.

It would be best not to treat the install wizard as a program in its own right but as a means to an end of installing the servers. As such, I think it should be represented as a server install option and put in the server box - when its out of tech preview status.

In terms of messaging, I try to never talk about the install wizard as if it is an exciting program in itself. I just keep calling it "the install wizard" to draw attention to the servers that it installs rather than the program itself. Because once you install the servers, I'm not sure if the users will keep it or uninstall it.

**Comment by balhar-jakub** [comment]
@1000TurquoisePogs 
1. I agree that with ZAC recommendation to release as Technical Preview, the download for now belong to the bottom as next Component on the same level as the three ones, that are already there. 
2. I am ok with putting it in the server section. I would however try to make it clear that the installation is on client side and your proposal of adding it as another installation methods makes sense to me.

File: download.md [hunk 4]
```diff
@@ -21,6 +21,10 @@
   .card-body:last-of-type {
     border-bottom: inherit;
   }
+  .right-box {
```
Comment (1000TurquoisePogs) [inline download.md:24]: unused? remove it then.

File: _data/releases.yml [hunk 5]
```diff
@@ -8,8 +8,9 @@ future:
     - version: 1.28.8
       release_date: 2024-10-14
 
-v2: 
-  - version: 2.16.0
+v2:
+  - zen_version: 1.0.0
```
Comment (1000TurquoisePogs) [inline _data/releases.yml:12]: version should be top. put zen_version below. it would confuse readers.

File: _includes/legal-script.html [hunk 6]
```diff
@@ -125,7 +125,56 @@
...
                 .get(
                     'package') + '.zip';
-        } else {
+        }
+        else if (params.has('type') && params.get('type') == "zen-windows") {
+            document.getElementById('download_button').href = "post_download.html?version=" + params.get('version') + "&type=" + params.get('type');
+            document.getElementById('download_button').onclick = function () {
+                ga && ga('send', 'event', 'download', 'zen-windows' + params.get('version'), 'zowe-enterprise-necessity-' + params.get(
+                    'version') + '.exe');
+                 window.open("https://zowe.jfrog.io/zowe/list/libs-release-local/org/zowe/zen/" + params.get('version') +
```
Comment (1000TurquoisePogs) [inline _includes/legal-script.html:134]: These look correct but I will await confirmation from @struga0258 once live before approving.

**Review by 1000TurquoisePogs — APPROVED** [review]

**Pull request merged**